packages/plugin-infra/src/types.ts

```diff
@@ -365,6 +365,7 @@ export namespace Plugin.SNSAdaptor {
         privileged_silentSign: () => (signer: ECKeyIdentifier, message: string) => Promise<PersonaSignResult>
         getPersonaAvatar: (identifier: ECKeyIdentifier | null | undefined) => Promise<string | null | undefined>
         ownProofChanged: UnboundedRegistry<void>
+        setPluginEnable: (id: string, enabled: boolean) => Promise<void>
```

Comment:
we have another concept of enabling/disabling which is not exposed to users. Do you mean: `setPluginMinimalMode`